pepper-apis/src/main/java/org/broadinstitute/ddp/event/dsmtask/api/DsmTaskData.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.ddp.event.dsmtask.api;
+
+/**
+ * Data which fetched from DsmTask pubsub message
+ * published by DSM to pubsub topic "dsm-to-dss-tasks"
+ */
+public class DsmTaskData {
```

Comment:
Can we remove `dsm` and replace with `PubSub`?  Technically anything can post stuff to pubsub, so the source might not necessarily be DSM.

Reply:
You mean to rename the topic or class name?

Reply:
Actually, good idea: so we can do more generic API for sending tasks to DSS..
@zyme what do you think to rename topics to:
"dsm-to-dss-task" to "dss-task"
"dss-to-dsm-task" to "dss-task-result" ?

Reply:
If we do it a more generic API then I suggest to add extra attribute "taskSource" (and in our case source="DSM")

Reply:
@zyme I replaced Dsm with PubSub